packages/playground/src/api/spatial/featuresAPI.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  BaseResourceAPI,
+  CogniteExternalId,
+  IdEither,
+  ExternalId,
+} from '@cognite/sdk-core';
+import { Features, FeaturesCreateItem, FeatureSearchFilter } from './types';
+
+export class FeaturesAPI extends BaseResourceAPI<Features> {
+  public create = (
+    featureTypeExternalId: CogniteExternalId,
+    features: FeaturesCreateItem[]
+  ): Promise<Features[]> => {
+    return this.createEndpoint<FeaturesCreateItem>(
+      features,
+      this.url(`${featureTypeExternalId}/features`)
+    );
+  };
+
+  public retrieve = (
+    featureTypeExternalId: CogniteExternalId,
+    ids: IdEither[],
+    queryParams?: { outputGeometryFormat: 'wkt' | 'geojson' }
```

Comment:
it feels like these should come from a type somewhere

Reply:
https://github.com/cognitedata/cognite-geospatial-sdk/blob/89b0eda90aab74086e47a9f9b993acfbf1f3e0aa/nodejs/src/types.ts#L24
I'll create a GeometryType like this one present in current geospatial-sdk.
Is that fine @rek ?